site/content/docs/_index.md

```diff
@@ -9,6 +9,10 @@ groups:
         link: /getting-started/
         desc: a good place to start if you are new to ZAP
     
+      - name: 'Getting Further Guides'
+        link: /docs/getting-further/
+        desc: in depth guides covering different ways you can use ZAP
+    
       - name: 'Desktop User Guide'
         link: /docs/desktop/
         desc: the help included with the ZAP desktop application 
```

Comment:
I'd suggest swapping this two. 

1. Started.
2. User guide.
3. Further.
4. Dev guide.

Reply:
I think the current order is better - the User Guide is not easy to read, its more of a reference guide we link to..

Reply:
Okay.